### ISSUE TITLE
Low quanlity output

### ISSUE THREAD

**Issue by thuyduong275** [lyuPang/CrossInitialization#3]
Hi, thankyou for your great work and for sharing this repo
I tried to run train_cross_init.py with my own image, but the generated results are not good (distorted faces, poor fidelity to the prompt, etc)
I would like to ask a few question to better understand how to get better results:

**My command to train:**
python train_cross_init.py \
    --save_steps 1000 \
    --only_save_embeds \
    --placeholder_token "<thuyduong220425>" \
    --train_batch_size 4 \
    --scale_lr \
    --n_persudo_tokens 2 \
    --reg_weight "1e-5" \
    --learning_rate 0.000625 \
    --max_train_step 3000 \
    --train_data_dir "./examples/input_images/GS_300dpi" \
    --celeb_path "./examples/wiki_names_v2.txt" \
    --pretrained_model_name_or_path "stabilityai/stable-diffusion-2-1-base" \
    --output_dir "./logs/Day220425/learned_embeddings" \
    --resolution 384 \
    --gradient_checkpointing \
    --gradient_accumulation_steps 2 

**My command to test:**
python test_cross_init.py \
    --pretrained_model_name_or_path "stabilityai/stable-diffusion-2-1-base" \
    --num_inference_steps 100 \
    --learned_embedding_path "./logs/Day220425/learned_embeddings/learned_embeds-steps-1000.bin" \
    --prompt "a {} person is eating his birthday cake" \
    --save_dir "./logs/Day220425/imagesOutput" \
    --num_images_per_prompt 10 \
    --n_iter 1

🙋‍♂️ My Questions:
- Do I need to adjust any learning rate, reg_weight or training steps in the scripts for better convergence?

![Image](https://github.com/user-attachments/assets/13c9a373-fc1c-4b52-bad5-3c61544ac772)

**Comment by lyuPang** [comment]
Thank you for trying out the repo and sharing your detailed setup! Here are some suggestions to improve your results:

1. Training Steps
The method in this repo is designed to converge efficiently with 320 training steps, not 3000. Please set `--max_train_step 320` to avoid overfitting and ensure optimal results.

2. Placeholder Token
The `--placeholder_token` must be a unique identifier (e.g., "<human1>", "<person1>") that does not exist in Stable Diffusion’s original vocabulary. Setting it to an empty string ("") or an existing token (e.g., "person") can confuse the model and degrade results.

3. Recommended Parameters
The README includes a "Training" section with tuned hyperparameters (e.g., learning rate, regularization weight). For consistency, please refer to those values first before further experimentation.

4. Model Limitations & Hyperparameter Tuning
Due to inherent biases in Stable Diffusion's training data, the method may struggle with certain facial features or prompts.
